### ISSUE TITLE
You can simply buy Hearthstones.

### ISSUE THREAD

**Issue by Cabro** [elysium-project/server#2305]
Innkeepers currently sell Hearthstones.
This is wrong, you should be given a new Hearthstone when you make an inn your home.
Hearthstones shouldn't be on the NPC's sell list.

**Comment by atiaran** [comment]
I am almost 100% sure that in retail it was also possible to buy a heartstone from an innkeeper. 
For example: situations that players "accidently" destroyed their HS they don't have to run back to the inn to set their HS again. Simply buy a new one at any inn and ready to go again.

**Comment by berillis** [comment]
No, you shouldn't be able to buy them, if you don't have hearthstone in your inventory/bank the innkeeper should have a gossip option "I've lost my hearthstone." or something like that and when clicking you would get a new hearthstone for free.
